src/hyperfiddle/rcf.cljc

```diff
@@ -18,6 +18,10 @@
 #?(:clj  (def ^:dynamic *enabled* (= "true" (System/getProperty "hyperfiddle.rcf.enabled")))
    :cljs (goog-define ^boolean ^:dynamic *enabled* false))
 
+(defn enable! [& [v]]
+  #?(:clj  (alter-var-root #'hyperfiddle.rcf/*enabled* (constantly (if (some? v) v true)))
+     :cljs (set! hyperfiddle.rcf/*enabled* (if (some? v) v true))))
```

Comment:
I have foolishly merged this without testing cljs because my env is broken and I can't imagine it doesn't work